src/ToDo.iOS/Entitlements.plist

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-	<dict>
-	</dict>
+<dict>
+	<key>keychain-access-groups</key>
+	<array>
+		<string>$(AppIdentifierPrefix)9TB54R6A6V.com.horus.unoplatform.todoapp</string>
```

Comment:
Please create a task in the issues list to replace this with the keychain group from uno team